media-api/app/lib/ImageResponse.scala

```diff
@@ -132,8 +153,11 @@ object ImageResponse {
     __.json.update(__.read[JsObject].map(_ ++ Json.obj("cost" -> cost.toString)))
   }
 
-  def addPersistedState(isPersisted: Boolean): Reads[JsObject] =
-    __.json.update(__.read[JsObject]).map(_ ++ Json.obj("persisted" -> isPersisted))
+  def addPersistedState(isPersisted: Boolean, persistenceReasons: List[String]): Reads[JsObject] =
+    __.json.update(__.read[JsObject]).map(_ ++ Json.obj(
+      "persisted" -> Json.obj(
+        "data" -> isPersisted,
```

Comment:
Sorry, me being silly - it's just info and has no actions / methods to be applied to is, so it shouldn't be a resource.